appscale/tools/agents/gce_agent.py

```diff
@@ -354,7 +354,6 @@ def create_ssh_key(self, parameters, all_ssh_keys):
       all_ssh_keys: A str that contains all of the SSH keys that are
         currently passed in to GCE instances.
     """
-    our_public_ssh_key = None
```

Comment:
Since this function is intended to add an SSH key that we just created, I think it would be better to let the exception from the read error bubble up rather than silently ignore the error and continue.